elytron-oidc-client/src/main/java/org/wildfly/extension/elytron/oidc/ProviderAttributeDefinitions.java

```diff
@@ -218,10 +236,39 @@ class ProviderAttributeDefinitions {
                     .setValidator(new StringLengthValidator(1, Integer.MAX_VALUE, true, true))
                     .build();
 
+    protected static final SimpleAttributeDefinition AUTHENTICATION_REQUEST_FORMAT =
+            new SimpleAttributeDefinitionBuilder(ElytronOidcDescriptionConstants.AUTHENTICATION_REQUEST_FORMAT, ModelType.STRING, true)
+                    .setAllowExpression(true)
+                    .setDefaultValue(new ModelNode("oauth2"))
+                    .setAllowedValues("oauth2", "request", "request_uri")
```

Comment:
Just a minor, we should make constants out of oauth2, request and request_uri . You can place them to ElytronOidcDescriptionConstants

Reply:
fixed!